packages/compass-components/src/components/with-status-marker.tsx

```diff
@@ -0,0 +1,117 @@
+import { palette } from '@leafygreen-ui/palette';
+import { css, keyframes } from '@leafygreen-ui/emotion';
+import { spacing } from '@leafygreen-ui/tokens';
+import React from 'react';
+
+export type StatusMarker =
+  | 'connected'
+  | 'disconnected'
+  | 'connecting'
+  | 'failed';
+export type StatusMarkerProps = {
+  status: StatusMarker;
+  children: React.ReactNode;
+};
+
+function ConnectedStatusMarker(): React.ReactElement {
```

Comment:
We usually don't put components in compass-components that are very specific and are only used by a single place in the whole application

Reply:
Well it's going to be used in different places in the sidebar, and it's a relatively small and simple, but I can move it to the sidebar.

Reply:
Small and simple is not a deciding factor for putting something in compass-components, using across multiple packages is